invenio_rdm_records/services/pids/tasks.py

```diff
@@ -9,8 +9,16 @@
 
 from celery import shared_task
 
+from invenio_rdm_records.proxies import current_rdm_records
+
 
 @shared_task(ignore_result=True)
-def register_pid(recid, pid_type):
-    """Register a PID."""
-    pass
+def register_pid(pid, scheme, record_id, provider_name=None):
+    """Registers a PID."""
+    record = current_rdm_records.records_service.record_cls.get_record(
```

Comment:
Can we do `service.read`? not to go that low to data access layer (i.e. `record_cls`)?